src/server/pkg/storage/track/pg_tracker.go

```diff
@@ -0,0 +1,222 @@
+package track
+
+import (
+	"context"
+	"database/sql"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+)
+
+var _ Tracker = &PostgresTracker{}
+
+type PostgresTracker struct {
+	db *sqlx.DB
+}
+
+func NewPostgresTracker(db *sqlx.DB) *PostgresTracker {
+	return &PostgresTracker{db: db}
+}
+
+func (t *PostgresTracker) CreateObject(ctx context.Context, id string, pointsTo []string, ttl time.Duration) error {
+	for _, dwn := range pointsTo {
+		if dwn == id {
+			return ErrSelfReference
+		}
+	}
+	return t.withTx(ctx, func(tx *sqlx.Tx) error {
+		var oid int
+		if err := func() error {
+			if ttl > 0 {
+				return tx.GetContext(ctx, &oid,
```

Comment:
The lowest ttl should probably be a second, and we should error similarly to the way we do through the renewal pfs api.

Reply:
I don't think we need to enforce that constraint at this level. Postgres supports intervals with resolution 1 microsecond.    I would argue only the system up a level can make reasonable assumptions about minimum latency.

Reply:
It is not really a big deal, I just cannot imagine needing a ttl less than a second for anything and having a check for anything less than that might help catch an error if something incorrectly sets the ttl.

Reply:
If we are going to have an arbitrary limit I think it should only be enforced in one place (right now the PFS driver). At this level the worst thing a small interval would cause is and object that expires before the caller can do anything with it. Also setting the ttl to a nanosecond could be useful for testing.

Reply:
Yeah, so if we had an internal call that set it for less than a second, then it is probably a bug that would get caught pretty obviously by a check for less than a second.

Reply:
Discussed offline, we will leave it as a TODO for now.